client/js/components/user/CustomerSettings/CustomerSettings.vue

```diff
@@ -311,39 +389,48 @@ export default {
     fetchCustomerData () {
       this.isLoadingSignLanguageOverviewVideo = true
       const payload = this.getRequestPayload()
-      dpApi.get(Routing.generate('api_resource_list', { resourceType: 'Customer' }), payload, { serialize: true })
-        .then((response) => this.handleCustomerResponse(response))
+      this.fetchCustomer(payload, { serialize: true }).then(() => {
+        this.isLoading = this.isLoadingSignLanguageOverviewVideo = false
+
+        /// /Update fields
+        const currentData = this.customerList[this.currentCustomerId].attributes
+        this.customerBrandingId = this.customerList[this.currentCustomerId].relationships?.branding?.data?.id
+
+        this.customer.imprint = currentData.imprint ? currentData.imprint : ''
+        this.customer.dataProtection = currentData.dataProtection ? currentData.dataProtection : ''
+        this.branding.logoHash = this.fileList[this.brandingList[this.customerBrandingId].relationships?.logo.data?.id]?.attributes.hash
+      })
```

Comment:
```suggestion
        const currentCustomer = this.customerList[this.currentCustomerId]
        const currentData = currentCustomer.attributes
        this.customerBrandingId = currentCustomer.relationships?.branding?.data?.id

        this.customer = {
          ...this.customer,
          dataProtection: currentData.dataProtection ? currentData.dataProtection : '',
          imprint: currentData.imprint ? currentData.imprint : '',
          logoHash: this.fileList[this.brandingList[this.customerBrandingId].relationships?.logo.data?.id]?.attributes.hash
          
        }
      })
```

Reply:
the last line with branding.logoHash I would like keep, because its another ressourcetype and another way, so stored in another data

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2212/commits/3074018497a56eaa721058f7bf9f48ce61dbabf0

Reply:
> the last line with branding.logoHash I would like keep, because its another ressourcetype and another way, so stored in another data

Right, my bad :+1: